app/dev-portal/dynamic-client-registration.md

```diff
@@ -46,14 +46,90 @@ faqs:
+{% endmermaid %}
+
+### What a valid DCR request and response look like
+
+**Request example:** Send a POST to the **Client Registration Endpoint** with client metadata as top-level JSON members. This example follows RFC 7591 and OIDC Dynamic Client Registration 1.0:
+
+```json
+{
+    "client_name": "Orders Web",
+    "redirect_uris": [
+        "https://app.example.com/callback"
+    ],
+    "grant_types": [
+        "authorization_code",
+        "refresh_token"
+    ],
+    "token_endpoint_auth_method": "client_secret_basic",
+    "application_type": "web"
+}
+```
+
+- `redirect_uris`, `grant_types`, `token_endpoint_auth_method`, and `client_name` are standard client metadata under RFC 7591.
+- `application_type` is an OIDC client metadata parameter with allowed values like web and native, from OIDC Dynamic Client Registration 1.0.
+
+**Response example:** On success, the endpoint returns HTTP 201 and a JSON body that contains the issued identifier and the registered metadata. When the server supports client management, it also returns a Registration Access Token and a Client Configuration Endpoint URI. An example of the response you might get:
+
+```json
+{
+    "client_id": "s6BhdRkqt3",
+    "client_secret": "ZJYCqe3GGRvdrudKyZS0XhGv_Z45DuKhCUk0gBR1vZk",
+    "client_id_issued_at": 1730419200,
+    "client_secret_expires_at": 0,
+    "registration_access_token": "this.is.an.access.token.value.ffx83",
+    "registration_client_uri": "https://server.example.com/connect/register?client_id=s6BhdRkqt3",
+    "token_endpoint_auth_method": "client_secret_basic",
+    "application_type": "web",
+    "redirect_uris": [
+        "https://app.example.com/callback"
+    ],
+    "client_name": "Orders Web",
+    "grant_types": [
+        "authorization_code",
+        "refresh_token"
+    ]
+}
+```
+
+- `client_id` is required. 
+- `client_secret` appears for confidential clients.
+- `client_id_issued_at` and `client_secret_expires_at` are standard response members.
+- `registration_access_token` and `registration_client_uri` appear when the server enables client management. Implementations return both values or neither.
+
+In a DCR strategy, the Dev Portal submits the registration request to your IdP and displays the returned client credentials to the developer’s application. This behavior appears in [Application authentication strategies](/auth-strategies).
 
 ## Authentication methods
 
-DCR support in {{site.konnect_short_name}} provides multiple methods by which applications can be authenticated using industry-standard protocols. These methods include:
+DCR registers the application as a client at your Identity Provider. The client authenticates when it requests tokens or calls protected resources after registration. DCR support in {{site.konnect_short_name}} provides multiple methods by which applications can be authenticated using industry-standard protocols. These methods include:
```

Comment:
```suggestion
DCR support in {{site.konnect_short_name}} provides multiple methods by which applications can be authenticated using industry-standard protocols. These methods include:
```
Since we already explained the how, I think we can leave this like this because this is covering more of the details about the different supported providers and methods.